libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -578,6 +578,11 @@ bool AP_Arming::rc_arm_checks(AP_Arming::Method method)
         check_failed(ARMING_CHECK_PARAMETERS, true, "Duplicate Aux Switch Options");
         check_passed = false;
     }
+    uint8_t flight_mode_channel;
+    if (rc().check_flight_mode_channel_conflict(flight_mode_channel)) {
+        check_failed(ARMING_CHECK_PARAMETERS, true, "FLTMODE_CH and RC%d_OPTION are in conflict", flight_mode_channel);
```

Comment:
I think we could shorten the message to, "FLTMODE_CH and RCx_OPTION conflict".  Also unfortunately Rover's parameter is just MODE_CH so we need to check the build here sadly.

Reply:
How about "Mode channel and RC%d_OPTION conflict"?

Reply:
@tatsuy, yes that sounds good to me. txs!